django/db/migrations/migration.py

```diff
@@ -184,15 +183,22 @@ def suggest_name(self):
         name to avoid VCS conflicts if possible.
         """
         name = None
-        if len(self.operations) == 1:
-            name = self.operations[0].migration_name_fragment
-        elif (
-            len(self.operations) > 1 and
-            all(isinstance(o, operations.CreateModel) for o in self.operations)
-        ):
-            name = '_'.join(sorted(o.migration_name_fragment for o in self.operations))
-        if name is None:
-            name = 'initial' if self.initial else 'auto_%s' % get_migration_name_timestamp()
+        if self.initial:
+            return 'initial'
```

Comment:
I've moved the initial migrations changes to a separate PR, see #14116.

Reply:
Acknowledged.
May you help me with the changes that I shall commit in [this PR](https://github.com/django/django/pull/14109)? Or Shall I just remove Conflicts(the `if condition` for the initial name)?